q2_itsxpress/_itsxpress.py

```diff
@@ -19,78 +19,42 @@
 
 """
 import os
+import pathlib
 import shutil
 import math
 
-import yaml
+import pandas as pd
 from q2_types.per_sample_sequences import (SingleLanePerSamplePairedEndFastqDirFmt,
                                            SingleLanePerSampleSingleEndFastqDirFmt,
-                                           FastqManifestFormat,
-                                           YamlFormat)
-from q2_types.per_sample_sequences._format import _SingleLanePerSampleFastqDirFmt
```

Comment:
Removing unsafe format usage --- this is a private format in q2-types, not intended for public consumption.